x-pack/plugins/observability/public/components/app/section/alerts/index.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiFlexGroup } from '@elastic/eui';
+import { EuiLink } from '@elastic/eui';
+import { EuiBadge } from '@elastic/eui';
+import { EuiFlexItem } from '@elastic/eui';
+import { EuiText } from '@elastic/eui';
+import { EuiIcon } from '@elastic/eui';
+import moment from 'moment';
+import { EuiHorizontalRule } from '@elastic/eui';
+import { SectionContainer } from '..';
+
+const _alerts = [
```

Comment:
What is this?

Reply:
already removed it... was using it as type.